### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by kingtvarshin**
Insuree filter applied and extension added and wiki urls added

### CODE REVIEW

File: api_fhir/converters/patientConverter.py [hunk 1]
```diff
@@ -1,18 +1,20 @@
...
+from api_fhir.models import Patient, AdministrativeGender, ImisMaritalStatus, Extension
 from api_fhir.models.address import AddressUse, AddressType
 from api_fhir.utils import TimeUtils, DbManagerUtils
 
-
 class PatientConverter(BaseFHIRConverter, PersonConverterMixin, ReferenceConverterMixin):
 
     @classmethod
     def to_fhir_obj(cls, imis_insuree):
+        # flag = views.InsureeViewSet.check_for_extension(views.InsureeViewSet,request)
```
Comment (areklalo) [inline api_fhir/converters/patientConverter.py]: Please remember to remove these comments if they are unnecessary

File: api_fhir/converters/patientConverter.py [hunk 2]
```diff
@@ -22,6 +24,8 @@ def to_fhir_obj(cls, imis_insuree):
         cls.build_fhir_marital_status(fhir_patient, imis_insuree)
         cls.build_fhir_telecom(fhir_patient, imis_insuree)
         cls.build_fhir_addresses(fhir_patient, imis_insuree)
+        # if flag:
```
Comment (areklalo) [inline api_fhir/converters/patientConverter.py]: Same as above

**Review by areklalo — CHANGES_REQUESTED** [review]

File: api_fhir/converters/patientConverter.py (hunk 1 above)
Comment (harsh-a1) [reply api_fhir/converters/patientConverter.py]: Done

**Review by areklalo — APPROVED** [review]

**Comment by areklalo** [comment]
If you decide that the PR is ready to merge then kindly change the status from draft to "Ready for review".

**Pull request merged**